docs/use_cases/user_gets_revoked.md

```diff
@@ -0,0 +1,19 @@
+# User gets revoked from a project #
+
+This happens whenever an user gets its permissions over the repository removed. This use case has a side case in where an user might not be able to see any project in the dashboard due to lack of permissions to any project.
+
+## Assumptions ##
+
+- User logged in.
+- Dashboard might show a particular list of 1 or more projects which the user has access to.
```

Comment:
We can move this to a `### Steps ###` section like:
1. If user has no create access, user sees message `...`
1. If user has create access, but no other repo access, user sees message `...`